bench/wtperf/wtperf.h

```diff
@@ -58,11 +58,15 @@ typedef struct __truncate_queue_entry TRUNCATE_QUEUE_ENTRY;
 #define	ZSTD_EXT							\
 	EXT_PFX EXTPATH "zstd/.libs/libwiredtiger_zstd.so" EXT_SFX
 
+#define	MAX_NUM_OF_MODIFICATIONS 16
+#define	MAX_SIZE_OF_MODIFICATION 10
```

Comment:
I'd suggest `MAX_MODIFY_NUM` and `MAX_MODIFY_PCT`. That way if there are more or defines for other operations they sort together based on the operation type. I'd add a comment about what these numbers means too.

Reply:
Changed as per your suggestion.